test/sqllogictest/transform/relax_must_consolidate.slt

```diff
@@ -1162,39 +1162,35 @@ FROM input;
 ----
 Explained Query:
   Return
-    Return
-      Union
-        ArrangeBy
-          input_key=[]
-          raw=true
-          Get::PassArrangements l1
-            raw=false
-            arrangements[0]={ key=[], permutation=id, thinning=(#0) }
-        Mfp
-          project=(#0)
-          map=(null)
-          Union consolidate_output=true
-            Negate
-              Get::Arrangement l1
-                project=()
-                key=
-                raw=false
-                arrangements[0]={ key=[], permutation=id, thinning=(#0) }
-            Constant
-              - ()
-    With
-      cte l1 =
+    Union
+      Get::PassArrangements l1
+        raw=true
+      Mfp
+        project=(#0)
+        map=(null)
+        Union consolidate_output=true
+          Negate
+            Get::Collection l1
+              project=()
+              raw=true
+          Constant
+            - ()
+  With Mutually Recursive
+    cte l1 =
+      ArrangeBy
+        input_key=[]
+        raw=true
         Reduce::Hierarchical
           aggr_funcs=[max]
           skips=[0]
-          monotonic
-          must_consolidate
+          buckets=[268435456, 16777216, 1048576, 65536, 4096, 256, 16]
```

Comment:
This looks weird, and .. no clue what to make of it. Looks like a worse plan for some reason.